### ISSUE TITLE
Ambiguities in Requirements from original source

### ISSUE THREAD

**Issue by manuelfuenmayor** [metanorma/ogc-GeoPose#4]
In relation to https://github.com/metanorma/ogc-GeoPose/pull/2.

Following is a list of ambiguities present in[ `clause_7_normative_text.adoc`](https://raw.githubusercontent.com/metanorma/ogc-GeoPose/geopose_encoding/standard/standard/clause_7_normative_text.adoc), in regards to Requirements:

### `/req/basic_ypr/sdu`

A Requirement Class and an individual Requirement have the is same label:

```adoc
[[req_class_basic_ypr_sdu]]
[requirements_class]
====
[%metadata]
label:: /req/basic_ypr/sdu
subject:: Implementation of Logical Model
description:: Basic-YPR Logical Model SDU
inherit:: /req/global
requirement:: /req/basic_ypr/sdu
requirement:: /req/tangent_plane_longitude/sdu
requirement:: /req/tangent_plane_latitude/sdu
requirement:: /req/tangent_plane_h/sdu
requirement:: /req/orientation_ypr_angles/sdu
====
````

```adoc
[[req_basic_ypr_sdu]]
[requirement]
====
[%metadata]
label:: /req/basic_ypr/sdu
description:: An implementation of a Basic-YPR Target SHALL consist of an
Outer Frame specified by an implicit WGS-84 CRS and an implicit
EPSG 4461-CS (LTP-ENU) coordinate system and explicit parameters to
define the tangent point. The Inner Frame SHALL be a rotation-only
transformation using Yaw, Pitch, and Roll angles specified in degrees
and represented as signed real number values.
====
````

### `/req/tangent_plane_longitude/sdu`

Descriptions are (slightly) different:

```adoc
[[req_tangent_plane_longitude]]
[requirement]
====
[%metadata]
label:: /req/tangent_plane_longitude/sdu
description:: An instance of a GeoPose tangentPoint.longitude attribute SHALL be
expressed as decimal degrees and represented as signed real number values.
====
````

```adoc
[requirement]
.B-TP-Lon-SDU
====
[%metadata]
label:: /req/tangent_plane_longitude/sdu
description:: An instance of a GeoPose tangentPoint.longitude attribute SHALL be
expressed as decimal degrees, represented as a signed real number.
====
````

### `/req/tangent_plane_latitude/sdu`

Descriptions are (slightly) different:

```adoc
[[req_tangent_plane_latitude]]
[requirement]
====
[%metadata]
label:: /req/tangent_plane_latitude/sdu
description:: An instance of GeoPose tangentPoint.latitude attribute SHALL be
expressed as decimal degrees and represented as signed real number values.
====
````

```adoc
[requirement]
.B-TP-Lat-SDU
====
[%metadata]
label:: /req/tangent_plane_latitude/sdu
description:: An instance of GeoPose tangentPoint.latitude attribute SHALL be
expressed as decimal degrees, represented as a signed real number.
====
````

### `/req/tangent_plane_h/sdu`

Descriptions are (quite) different.

```adoc
[[req_tangent_plane_h]]
[requirement]
====
[%metadata]
label:: /req/tangent_plane_h/sdu
description:: An instance of a GeoPose tangentPoint.h attribute SHALL be
expressed as a height in meters above the WGS-84 ellipsoid and represented
as a signed real number value.
====
````

```adoc
[requirement]
.B-TP-h-SDU
====
[%metadata]
label:: /req/tangent_plane_h/sdu
description:: An instance of a GeoPose tangentPoint.h attribute SHALL be
expressed as a height in meters above the WGS-84 ellipsoid, represented
as a signed real number. If the tangent point is above the WGS-84 ellipsoid,
the value SHALL be positive. If the tangent point is below the WGS-84
ellipsoid, the value SHALL be negative.
====
````

### `/req/time/geopose_instant/sdu`

Names are different:

```adoc
[[req_pose_geopose_instant_sdu]]
[requirement]
.GP-Instant-SDU
====
[%metadata]
label:: /req/time/geopose_instant/sdu
description:: The Logical Model attribute GeoPoseInstant SHALL be
expressed in Unix Time in seconds multiplied by 1,000. The unit of measure SHALL be milliseconds.
====
````

```adoc
[requirement]
.Graph-SDU
====
[%metadata]
label:: /req/time/geopose_instant/sdu
description:: The Logical Model attribute GeoPoseInstant SHALL be
expressed in Unix Time in seconds multiplied by 1,000. The unit of measure SHALL be milliseconds.
====
````

### `/req/frame_specification/id/sdu`

There is a similar label that has the same content, name and description: `/req/frame_specification_id/sdu`

### `/req/frame_specification/parameters/sdu`

Descriptions are different:

```adoc
[[req_frame_specification_parameters_sdu]]
[requirement]
.FS-Parameters-SDU
====
[%metadata]
label:: /req/frame_specification/parameters/sdu
description:: The frame_specification.parameter attribute SHALL contain all
parameters needed for the corresponding authority and ID. The definition
of these parameters is outside the scope of GeoPose.
====
````

```adoc
[requirement]
.FS-Parameters-SDU
====
[%metadata]
label:: /req/frame_specification/parameters/sdu
description:: The frame_specification.parameter attribute SHALL contain all
parameters needed for the corresponding authority and ID.
====
````

**Comment by ronaldtse** [comment]
@manuelfuenmayor thank you for listing out the issues! This is very helpful especially since there are people implementing the spec right now. 

@ogcscotts @ghobona should we directly raise these issues with the editor? Thanks.

**Comment by ghobona** [comment]
@manuelfuenmayor @ronaldtse @3DXScape I will suggest some modifications to some of the identifiers. I am preparing a crosswalk (mappings on a spreadsheet) from current identifier to suggested identifier, for each requirements class, requirement, conformance class, and abstract test. I will share it with you by 2022-08-14.

**Comment by ghobona** [comment]
@manuelfuenmayor @ronaldtse, I have agreed with the Editor on the mappings/crosswalk. I will complete the editing of the document.

Please proceed to merge the changes from [metanorma/ogc-GeoPose/tree/geopose_encoding](https://github.com/metanorma/ogc-GeoPose/tree/geopose_encoding) to [metanorma/ogc-GeoPose/tree/main](https://github.com/metanorma/ogc-GeoPose/tree/main). Once you have merged the [PR#2](https://github.com/metanorma/ogc-GeoPose/pull/2),  please proceed to create another PR for updating [opengeospatial/GeoPose/tree/main](https://github.com/opengeospatial/GeoPose/tree/main).

**Comment by ronaldtse** [comment]
Closing this as we have an updated table from @ghobona.
